packages/grid/x-data-grid-pro/src/models/gridGroupingColDefOverride.ts

```diff
@@ -15,7 +15,7 @@ export interface GridGroupingColDefOverride<R extends GridValidRowModel = any>
    * The field from which we want to apply the sorting and the filtering for the grouping column.
    * It is only useful when `props.rowGroupingColumnMode === "multiple"` to decide which grouping criteria should be used for sorting and filtering.
    * Do not have any effect when building the tree with the `props.treeData` feature.
-   * @default: The sorting and filtering is applied based on the leaf field in any, otherwise based on top level grouping criteria.
+   * @default The sorting and filtering is applied based on the leaf field in any, otherwise based on top level grouping criteria.
```

Comment:
this is not the intended usage of the `@default` directive. @cherniavskii do we use it that way in the codebase very often?

Reply:
I guess I could also answer this question. I've found yesterday many incorrect usages of the @default directive some have syntax problems and some incorrect usages of that directive, I could try make a more global PR and an issue to fix it, sounds good?

Reply:
It's used in a similar way in the Pickers codebase a few times as well. 🤔 
Maybe it's not the most correct usage, but it works and helps us to build the API documentation that would be useful. 🙈

Reply:
@Lev-Shapiro thanks, but that's not what I intended with that comment. I know it is not the correct usage of it, but my intention was to know if we use it that way elsewhere or if it is the only time.

As mentioned by @LukasTy it does not pose any threat and if it is a valuable tool we should continue using it like this! 💯